e2e/cypress/integration/archived_channel/archive_channel_post_spec.js

```diff
@@ -47,4 +57,71 @@ describe('Archived channels', () => {
         // * RHS text box should not be visible
         cy.get('#reply_textbox').should('not.be.visible');
     });
+
+    it.only('MM-T1722 Can click reply arrow on a post from archived channel, from saved posts list', () => {
+        let testArchivedChannel;
+        let postId;
+        let permalink;
+
+        cy.apiAdminLogin();
+
+        // # Create a channel that will be archived
+        cy.apiCreateChannel(testTeam.id, 'archived-channel', 'Archived Channel').then(({channel}) => {
+            testArchivedChannel = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${testArchivedChannel.name}`);
+
+            // # Post message
+            cy.postMessage('Test');
+
+            // # Create permalink to post
+            cy.getLastPostId().then((id) => {
+                postId = id;
+                permalink = `${Cypress.config('baseUrl')}/${testTeam.name}/pl/${postId}`;
+
+                // # Click on ... button of last post
+                cy.clickPostDotMenu(postId);
+
+                // # Click on "Copy Link"
+                cy.uiClickCopyLink(permalink);
+
+                // # Post the message in another channel
+                cy.get('#sidebarItem_off-topic').click();
+                cy.postMessage(`${permalink}`).wait(TIMEOUTS.ONE_SEC);
+            });
+
+            // # Archive the channel
+            cy.visit(`/${testTeam.name}/channels/${testArchivedChannel.name}`);
+            cy.uiArchiveChannel();
+        });
+
+        // # Change user
+        cy.apiLogout();
+        cy.reload();
+        cy.apiLogin(otherUser);
+        cy.visit(`/${testTeam.name}/channels/off-topic`);
+
+        // # Read the message and save post
+        cy.get('a.markdown__link').click();
+        cy.getNthPostId(1).then((_postId) => {
+            cy.clickPostSaveIcon(_postId);
+        });
+
+        // # View saved posts
+        cy.get('#channelHeaderFlagButton').click();
+        cy.wait(TIMEOUTS.HALF_SEC);
```

Comment:
Do we need explicit wait?

Reply:
Yes, otherwise the test is flaky and sometimes failed at least for me.